docs/native/src/identity/specifytokenissuance.rst

```diff
@@ -1,3 +1,27 @@
 
-|stub-icon| Specifying Token Issuance URL
-=========================================
+Post-Authorization token endpoint
+=================================
+
+The token endpoint URL (:rfc:`6749#section-3.2`) is normally obtained from the initial unauthenticated call 
+described at :ref:`ClientBootstrapper`.
+
+If the token endpoint URL cannot be determined before the end-user has completed the sign-in process, 
+an alternative token endpoint URL may be supplied.
+
+This is done via a tk= URL parameter appended to the value of the
```

Comment:
Recommend wrapping the tk= portion in backticks so it renders like this: `tk=`

E.g.
```
``tk=``
```